Gruntfile.js

```diff
@@ -58,6 +58,24 @@ module.exports = function(grunt) {
 						]
 					}
 				}
+			},
+			rtlcss: {
+				o2: {
+					src: 'css/style.css',
+					dest: 'css/rtl/style-rtl.css'
+				},
+				modules: {
+					files: {
+						'modules/checklists/css/rtl/style-rtl.css':    'modules/checklists/css/style.css',
```

Comment:
There is. My initial research led me to the conclusion that it [wasn't necessarily the way to go](http://stackoverflow.com/questions/16668783/dynamic-file-mapping-with-relative-destination), but I can definitely automate it so that new modules can just be dropped in. I'm going to go ahead and do the automation because it sounds like an interesting challenge.